cli/colors.rs

```diff
@@ -10,10 +10,11 @@ use termcolor::{Ansi, ColorSpec, WriteColor};
 use termcolor::{BufferWriter, ColorChoice};
 
 lazy_static! {
-        // STRIP_ANSI_RE and strip_ansi_codes are lifted from the "console" crate.
-        // Copyright 2017 Armin Ronacher <armin.ronacher@active-4.com>. MIT License.
+        // The previous ANSI pattern didn't work
+        // this one is from Jeff on StackOverflow.
+        // https://stackoverflow.com/a/33925425
```

Comment:
How did it not work? It would be great to have an example that we could put in the unit tests.

Reply:
Oh. Well I tried it and the stripColor didn't work... Let me see if I can put it to the test.

Reply:
Oh, it's the rust pattern! The previous pattern that didn't work was in the `colors.ts` file, so what I did is to stay consistent and replace this regex as well. I guess I have to modify the comment.